src/main/resources/templates/tags/create.html

```diff
@@ -33,6 +33,8 @@
                     <textarea type="text" class="form-control" id="description" name="description"></textarea>
                 </div>
                 <button type="submit" class="btn btn-primary float-right">Submit</button>
+                <button type="button" class="btn btn-primary float-right mr-2" onclick="window.location.href='/tags'" >Back</button>
```

Comment:
This commit is also old as it utilizes absolute address vs. relative address.

Reply:
Seems to be up to date with master.  Is it on a different branch?

Reply:
Yes its on the CreateBackButtons branch.

Reply:
It shouldn't be an issue, if we pull this and then the second branch the end result will be the same

Reply:
Gotcha, should be merged in now